packages/angular/package.json

```diff
@@ -29,7 +29,8 @@
     "build": "npx ng build",
     "watch": "npx ng build --watch --configuration development",
     "test": "npx ng test",
-    "generate": "rollup -c; node .autogen.cjs; rm .autogen.cjs"
+    "generate": "rollup -c; node .autogen.cjs; rm .autogen.cjs",
+    "gallery:angular": "ng serve angulargalleryview"
```

Comment:
Since we're already inside the `Angular` package, I don't think there's a need for the `:angular` suffix here.